packages/icons/src/disable.svg

```diff
@@ -0,0 +1,5 @@
+<svg fill="none" height="16" viewBox="0 0 17 16" width="17" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Is this a toggle?

Reply:
I see height here. Is it required?
https://harness.atlassian.net/wiki/spaces/PLATFORM/pages/1963262066/SVG+Guidelines+Best+Practices

Reply:
yes

Reply:
Shouldn't toggle be interactive rather than icon?

Reply:
good callout. Updated!

Reply:
<img width="213" alt="image" src="https://user-images.githubusercontent.com/20707504/200246888-9af5ac12-df83-49ba-93f3-519623298c30.png">


It's needed here

Reply:
<img width="175" alt="image" src="https://user-images.githubusercontent.com/20707504/200247101-1a48ef41-624e-4822-902e-07b318402b5b.png">

Reply:
Got it. Thank you